README.md

```diff
@@ -1 +1,79 @@
-# Meals-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Meal App
+
+> Meal App is an app that shows 3 different categories of food. In each category there are 6 different meals within that category. Meal DB API provides the data about categories and all the meals. Meal App has homepage and comment modal section. App allows user to like and comment.
+
+[Click here for LIVE DEMO](https://faizi2500.github.io/Meals-App/dist/)
+
```

Comment:
- [ ] Kindly, fix the live link demo, it opens the readme file as well instead of live demo link.